tools-local/unix/cmake/get-version.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Gets the version of CMake at the specified path.
+
+usage()
+{
+    echo "Usage: $0 cmake ToolName --ToolPath <path>"
```

Comment:
This applies throughout the scripts.

---
In reply to: [109680711](https://github.com/ravimeda/corefx/pull/7#discussion_r109680711) [](ancestors = 109680711)